server/belga/publish/belga_newsml_1_2.py

```diff
@@ -868,7 +868,17 @@ def _format_newslines(self, newscomponent_2_level, item):
         SubElement(newslines, 'DateLine')
         SubElement(newslines, 'HeadLine').text = item.get('headline')
         SubElement(newslines, 'CopyrightLine').text = item.get('copyrightholder')
-        SubElement(newslines, 'CreditLine').text = self.DEFAULT_CREDITLINE
+
+        #SDBELGA-672
+        for subject in item.get('subject', []):
+            if subject.get('scheme') == 'sources':
+                SubElement(newslines, 'CreditLine').text = subject.get("name")
+
+        if len(newslines.xpath('CreditLine')) == 0:
+            if item.get('source'):
+                SubElement(newslines, 'CreditLine').text = item['source'].strip("incoming")
```

Comment:
it should handle both `incoming` and `Incoming`, they are not consistent there. and also remove the space between name and incoming pls